### PR TITLE
Add VSCode module to scaffolder

### PR DESCRIPTION
**Pull request by aholmes**
This PR adds a module to the scaffolder for users working with VSCode. Currently this just adds a launch.json with a couple debugger configurations.

### CODE REVIEW

File: src/web/Ligare/web/scaffolding/__main__.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ class ScaffoldParsedArgs(NamedTuple):
     name: Operation
     endpoints: list[Operation]
     template_type: Literal["basic", "openapi"]
-    modules: list[Literal["database", "test"]] | None
+    modules: list[str] | None
```
Comment (dan-knight) [inline src/web/Ligare/web/scaffolding/__main__.py:38]: Is this change directly related to the VScode module?

**Review by dan-knight — APPROVED** [review]

File: src/web/Ligare/web/scaffolding/__main__.py (hunk 1 above)
Comment (aholmes) [reply src/web/Ligare/web/scaffolding/__main__.py:38]: It is more related to the ability to create more Scaffold modules without having to update the `modules` type or its values. We don't lose anything in practice; it has always been a list of strings, but now the specific values are unimportant to the type checker because they are dynamically set.

**Review by dan-knight — APPROVED** [review]
